spec/unit/analytics/ga4_import/elastic_search_relevancy_serialiser_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+
+RSpec.describe Analytics::Ga4Import::ElasticSearchRelevancySerialiser do
+  describe "#relevance" do
+    let(:consolidated_data) { { "/example" => 40, "/example2" => 20, "/example3" => 100, "/example3/child" => 100 } }
```

Comment:
This should be an array rather than a hash, though I suggest you set individual ones for the tests in question for easier readability.